package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-dialog-hook",
-  "version": "0.0.8",
+  "version": "0.1.0",
```

Comment:
IMO this isn't major change, we can use semantic-changelog btw

Reply:
Will change to 0.0.9